SneakPeek/src/com/cyscorpions/dalejulian/sneakpeek/adapters/SneakerCategoryAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.cyscorpions.dalejulian.sneakpeek.adapters;
+
+import java.util.ArrayList;
+
+import com.cyscorpions.dalejulian.sneakpeek.R;
+import com.cyscorpions.dalejulian.sneakpeek.R.id;
+import com.cyscorpions.dalejulian.sneakpeek.models.SneakerCategory;
+
+import android.app.Activity;
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+public class SneakerCategoryAdapter extends ArrayAdapter<SneakerCategory> {
+	private Context mContext;
+	private int layoutResourceId;
+	// SneakerCategory[] data = null;
+	private ArrayList<SneakerCategory> data;
+
+	// view holder
+	static class CategoryHolder {
+		TextView categoryName, categoryDesc;
+	}
+
+	public SneakerCategoryAdapter(Context appContext, int layoutResourceId,
+			ArrayList<SneakerCategory> data) {
+		super(appContext, layoutResourceId, data);
+		this.mContext = appContext;
+		this.layoutResourceId = layoutResourceId;
```

Comment:
Can I have specifics po regarding this matter? Because I am passing the layout resource ID po from MainActivity.java. Should I change this po so that the layout resource ID should be in the Adapter's getView() method instead of passing it from MainActivity? Thank you po.